app/helpers/sessions_helper.rb

```diff
@@ -0,0 +1,39 @@
+module SessionsHelper
```

Comment:
I read it, but since the articles are only arguments, it would be easiest if you give me an example of should I refactor that, because I have no idea how to proceed like this.

Reply:
Hey, sure.
I did explain.
It's a matter of where code should be put so those not related to views specifically should not be in helpers but go directly to contoller etc.
Helpers are automatically available to views because it is made for views.
Does this make sense?

Reply:
Yes, it is, you said exactly what articles are talking about, but there is still no example of how to apply this right now. I mean, where should I put the code from session_helper?

Reply:
Any method that isnt specifically for viee like i mentioned could be added directly to application controller.
Like i said above.
Those that are view-specific can stay. 
Remove the include in application controller.

Reply:
Do you misunderstand me? I actually explained it in first comment.
I even stated which method could stay.
I am really confused 😅

Reply:
It's about organization of code. The way it was intended.
It is a requirement for all rails projects

Reply:
if I move the code from helper to controller, the methods like "logged_in?" won't work in views like posts, I know, I am trying it

Reply:
I think I did it